src/modules/[chain]/staking/index.vue

```diff
@@ -82,24 +82,35 @@ const changes = computed(() => {
     return changes;
 });
 
-const change24 = (key: Key) => {
-    const txt = key.key;
+const change24 = (entry: { consensus_pubkey: Key; tokens: string }) => {
+    const txt = entry.consensus_pubkey.key;
     // const n: number = latest.value[txt];
     // const o: number = yesterday.value[txt];
     // // console.log( txt, n, o)
     // return n > 0 && o > 0 ? n - o : 0;
-    return changes.value[txt];
+
+    const latestValue = latest.value[txt];
+    if (!latestValue) {
+        return;
+    }
+
+    const displayTokens = format.tokenAmountNumber({
+        amount: parseInt(entry.tokens, 10).toString(),
+        denom: staking.params.bond_denom,
+    });
+    const coefficient = displayTokens / latestValue;
```

Comment:
@freak12techno not sure if we need round `coefficient` here.

Let's merge for now.  if it's inaccurate, we can improve it later

Reply:
Likely yes, as in practice I've seen coefficient being not 1 but something like 1.000033 or so, not sure what causes it (maybe token value rounding?), but I doubt this coefficient might ever be a float instead of int.
I can fix it in a separate PR if needed, let me know if it is needed.